source/js/buyers-guide/bg-main.js

```diff
@@ -16,64 +17,79 @@ import ProductGA from "./product-analytics.js";
 // until all the React stuff is _actually_ done.
 const apps = [];
 
+let env, networkSiteURL, csrfToken;
+
 let main = {
   init() {
-    ReactGA.initialize(`UA-87658599-6`);
-    ReactGA.pageview(window.location.pathname);
+    this.fetchEnv(envData => {
```

Comment:
Mimicked what we have in `main.js`. 

@Pomax (probably as a followup). Should we think of a way to deal with site-wide shared JS component (like footer) instead of duplicating code in both `main.js` and `bg-main.js`?

Reply:
Yeah, if you want to file an issue around making things that can be shared, shared, we can do some investigations there.

Reply:
Follow-up ticket filed: https://github.com/mozilla/foundation.mozilla.org/issues/3055